CMakeLists.txt

```diff
@@ -1,21 +1,23 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6.2)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0 FATAL_ERROR)
```

Comment:
oldest cmake depends on the distributions you want to support. right now it breaks the build for sle12. but we can work around by building a newer cmake in the nitrokey project to keep building it.